### ISSUE TITLE
[$250] Workspace - Member WS chat LHN shows "closed this report" after inviting the member again

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#42567]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.75-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Admin: Log in with a new expensifail account
2. Admin: Create a workspace
3. Admin: Invite a Gmail member
4. Member: Log in
5. Admin: Remove the member
6. Admin: Invite the member again after the WS room archived for the member
7. Member: Open the WS room from the LHN
8. Member: Open any chat or room from the LHN
9. Member: Open the WS room from the LHN again

## Expected Result:
It should show "No activity yet".

## Actual Result:
Member WS chat LHN shows "closed this report" after inviting the member again.

## Workaround:
n/a

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/6348092c-15c2-4d24-8e24-67969c1e265f

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01460d4e90751b2f8f</li>
        <li>Upwork Job ID: 1794062074032300032</li>
        <li>Last Price Increase: 2024-05-24</li>
    </ul>
</details>

**Comment by kbecciv** [comment]
We think that this bug might be related to #wave-collect - Release 1

**Comment by kbecciv** [comment]
@zanyrenney FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by tsa321** [comment]
Backend issue, the server response for the report:
```
lastActionType : "CLOSED"
lastMessageHtml : "closed this report"
lastMessageText : "closed this report"
```

**Comment by DylanDylann** [comment]
@zanyrenney This is a BE bug, could you add the internal label to get an eye from the internal engineer?

**Comment by kadiealexander** [comment]
Added `internal` label.

**Comment by kadiealexander** [comment]
Not overdue! Waiting on an engineer to pick this up. I don't think it's high enough priority to increase the price.

**Comment by kadiealexander** [comment]
Handing back to Zany since she's back!

**Comment by zanyrenney** [comment]
chnaging the priority of this to weekly as i don't think its high enough priority to be daily.

**Comment by DylanDylann** [comment]
@zanyrenney I think we should move this issue to monthly 😄

**Comment by MariaHCD** [comment]
👋🏼 Just noting that I noticed this bug on our test workspace. I was confused by the messaging in the LHN.

![Screenshot 2024-09-09 at 10 20 51 AM](https://github.com/user-attachments/assets/7be7e10b-19df-49de-b55e-22c9597a1263)

**Comment by zanyrenney** [comment]
We need an internal engineer on this one @MariaHCD so it would be awesome if you had capacity to help if you've also experienced this. Let me know / feel free to assign yourself!

**Comment by zanyrenney** [comment]
lookingfor an internal engineer

**Comment by zanyrenney** [comment]
bump @MariaHCD do you think you might be able to help on this one?

**Comment by zanyrenney** [comment]
bump @MariaHCD

**Comment by zanyrenney** [comment]
Still waiting on an internak engineer.

**Comment by zanyrenney** [comment]
Moving to expense.

**Comment by zanyrenney** [comment]
I asked Maria in DM if she might have capacity to pick this up, she isn't able to.

**Comment by zanyrenney** [comment]
https://expensify.slack.com/archives/C06ML6X0W9L/p1737129412279789

**Comment by zanyrenney** [comment]
This has been open for over six months, I'm gonna consider this not a priority enclose it out. If we still can experience many users who are having this problem, or it comes up in the full story sessions from the research team, we can reopen this issue but currently it doesn't seem like this is a priority to resolve and seems very hard to reproduce.
